LAMBO token submission

billpi1 opened the pull request:
# Pull Request Template

## Description

LAMBO Registration request :)

## Type of change

- [ ] Metadata related change
- [ ] Other

## Checklist:

- [ ] For metadata related changes, this PR code passes the GitHub Actions metadata validation


## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

Kartiiyer12 commented:
Hi @billpi1 Could you please provide more information about the utility of the token. From the name and the logo, its seems you are impersonating Lamborghini  and the descriptions says "Available to use in Lambo utilities stores in Barcelona". 
Can you please elaborate?

billpi1 commented:
> Hi @billpi1 Could you please provide more information about the utility of the token. From the name and the logo, its seems you are impersonating Lamborghini and the descriptions says "Available to use in Lambo utilities stores in Barcelona". Can you please elaborate?

Hi
I changed desc  because it got confusing about how and where exactly however i supposed to use token in own store . Anyway i changed desc to focus on online products like as NFTs

billpi1 commented:
change
